docs/turn-howto.md

```diff
@@ -96,10 +109,28 @@ This will install and start a systemd service called `coturn`.
         # TLS private key file
         pkey=/path/to/privkey.pem
 
+    In this case, replace the `turn:` schemes in the `turn_uri` settings below
+    with `turns:`.
+
+    We recommend that you only try to set up TLS/DTLS once you have set up a
+    basic installation and got it working.
+
 1.  Ensure your firewall allows traffic into the TURN server on the ports
-    you've configured it to listen on (By default: 3478 and 5349 for the TURN(s)
-    traffic (remember to allow both TCP and UDP traffic), and ports 49152-65535
-    for the UDP relay.)
+    you've configured it to listen on (By default: 3478 for TURN and 5349 for
+    TURNs traffic (remember to allow both TCP and UDP traffic), and ports
```

Comment:
A typo, I think?

```suggestion
    TURNS traffic (remember to allow both TCP and UDP traffic), and ports
```

Reply:
it was somewhat deliberate. I should probably rephrase though.

Reply:
hopefully better now?

Reply:
Looks fine to me!